examples/bigcommerce/components/cart.tsx

```diff
@@ -42,74 +121,23 @@ export function ProductCart({ className }: ProductCartProps) {
         </Transition>
       </Popover.Button>
       <Transition
-        className={'absolute z-10'}
+        className={'fixed left-5 right-5 sm:left-[unset] sm:right-[unset] sm:absolute z-10'}
```

Comment:
mobile styles for the cart to look like:
<img width="454" alt="CleanShot 2022-08-29 at 11 20 25@2x" src="https://user-images.githubusercontent.com/20950876/187247754-ee66d402-f1da-43e1-82e8-50671df4a600.png">